packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -116,10 +137,200 @@ export default class VTable {
+                        }
+                    });
+                });
+                break;
+            case TableBorderFormat.onlyExternalHeaderRowAndFirstColumnBorders:
+                this.cells.forEach(row => {
+                    row.filter(cell => cell.td).forEach(cell => {
+                        if (this.cells.indexOf(row) !== 0) {
+                            cell.td.style.borderTopColor = TRANSPARENT;
+                        }
+
+                        if (
+                            this.cells.indexOf(row) !== this.cells.length - 1 &&
+                            this.cells.indexOf(row) !== 0
+                        ) {
+                            cell.td.style.borderBottomColor = TRANSPARENT;
+                        }
+                        if (row.indexOf(cell) !== 0) {
+                            cell.td.style.borderLeftColor = TRANSPARENT;
+                        }
+                        if (
+                            row.indexOf(cell) !== this.cells[0].length - 1 &&
+                            row.indexOf(cell) !== 0
+                        ) {
+                            cell.td.style.borderRightColor = TRANSPARENT;
+                        }
+                        if (cell === this.cells[0][0]) {
+                            cell.td.style.borderRightColor = TRANSPARENT;
+                        }
+                    });
+                });
+                break;
+            case TableBorderFormat.removeHeaderRowMiddleBorder:
+                this.forEachCellOfRow(0, (cell, i) => {
+                    if (i !== 0) {
+                        cell.td.style.borderLeftColor = TRANSPARENT;
+                    }
+                    if (i !== this.cells[0].length - 1) {
+                        cell.td.style.borderRightColor = TRANSPARENT;
+                    }
+                });
+                break;
+            case TableBorderFormat.onlyMiddleBorders:
+                this.forEachCellOfColumn(0, (cell, row, i) => {
+                    if (cell.td) {
+                        cell.td.style.borderLeftColor = TRANSPARENT;
+                    }
+                });
+                this.forEachCellOfColumn(this.cells[0].length - 1, (cell, row, i) => {
+                    if (cell.td) {
+                        cell.td.style.borderRightColor = TRANSPARENT;
+                    }
+                });
+        }
+    }
+
+    /**
+     * Apply custom design to the first table column
+     * @param format
+     * @returns
+     */
+    private setFirstColumnFormat(format: Partial<TableFormat>) {
+        if (!format.firstColumn) {
+            this.forEachCellOfColumn(0, (cell, row, i) => {
+                if (cell.td) {
+                    cell.td = changeElementTag(
+                        cell.td,
+                        TABLE_CELL_TAG_NAME
+                    ) as HTMLTableCellElement;
+                    cell.td.scope = '';

```

Comment:
What is this line for?

Reply:
To remove the scope, when remove first row or first column.